tsconfig.json

```
@@ -7,6 +7,6 @@
     "strictNullChecks": false,
     "sourceMap": true,
     "noImplicitAny": false,
```

Это правило всегда должно быть в true
```suggestion
    "noImplicitAny": true,
```